spec/requests/confirmation_spec.rb

```diff
@@ -5,4 +5,19 @@
     get user_confirmation_path(confirmation_token: user.confirmation_token)
     assert_enqueued_jobs 1, only: ActivateEmailSubscriptionsJob
   end
+
+  context "with a logged-in user" do
```

Comment:
So perhaps we could add a feature / integration to follow the link clicking through?